android/src/main/java/com/liferay/mobile/android/auth/basic/CookieAuthentication.java

```diff
@@ -17,20 +17,35 @@
 import com.liferay.mobile.android.http.Request;
 
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
 
 /**
  * @author Javier Gamarra
  */
 public class CookieAuthentication extends BasicAuthentication {
 
+	public final static int COOKIE_EXPIRATION_TIME = 15 * 60;
```

Comment:
why public?

Reply:
Mainly to use it from Screens and not define the same constant there